charts/distributor-devicemanager/values.yaml

```diff
@@ -0,0 +1,37 @@
+namespace: "hanwha"
+fullnameOverride: distributor-devicemanager
+
+image:
+  replicaCount: 1
+  repository: ghcr.io/htaic/distributor-devicemanager-local
+  pullPolicy: Always
+  tag: "latest"
+  hostname: distributor-devicemanager
+  containerPort: 3303
+  env:
+    - name: REDIS_PORT
+      value: "6379"
+    - name: REDIS_HOST
+      value: baseline-redis
```

Comment:
these actually exist in the environment's secrets already... it is the baseline-redis-secret (see the envFrom in the complete-auth-service helm chart)